### PR TITLE
Apply/reedem shares error display

### LINKED ISSUE #2670: Apply/Redeem Shares
1.Create share product
2.Apply to a client 
3.Apply Additional Share OR Redeem share
Submit Form without filling any field  >> There should be a thrown error since **Request Date** and **Total Number of Shares** are mandatory 
When you empty **Request Date** there is no thrown error but when you empty  **Total Number of Shares** there is a thrown error 
ras

### PR DESCRIPTION
**Pull request by ankit01ojha**
fixed #2670

## Description
Stopped the submission of the form. Added the validation to it.

## Related issues and discussion
#2670

### CODE REVIEW

**Comment by mbj36** [comment]
Thanks for the PR, It will be reviewed Shortly!

(Powered by [GitMate.io](https://gitmate.io))

**Comment by ankit01ojha** [comment]
@gkrishnan724 , modified !

File: app/views/shares/shareaccountactions.html [hunk 1]
```diff
@@ -45,9 +45,12 @@ <h4>{{ title | translate}}</h4>
                                 class="required">*</span></label>
 
                         <div class="col-sm-3">
-                            <input id="{{modelName}}" type="text" sort datepicker-pop="dd MMMM yyyy"
-                                   ng-model="formData[modelName]" is-open="opened" min="minDate" max="restrictDate" class="form-control"/>
+                            <input id="{{modelName}}" type="text" name="{{modelName}}" sort datepicker-pop="dd MMMM yyyy"
```
Comment (gkrishnan724) [inline app/views/shares/shareaccountactions.html]: dont put the name as `{{modelName}}` keep it something like `DateField`

**Comment by ankit01ojha** [comment]
@gkrishnan724 , done.

**Pull request merged**